schemas/semconv.schema.json

```diff
@@ -41,6 +41,48 @@
 		}
 	},
 	"$defs": {
+		"Deprecated": {
```

Comment:
could you please also update https://github.com/open-telemetry/weaver/blob/main/schemas/semconv-syntax.md - an informal and reader-friendly version of it?